web_responsive/__manifest__.py

```diff
@@ -13,7 +13,7 @@
     "license": "LGPL-3",
     "installable": True,
     "depends": [
-        'web',
+        'web_editor',
```

Comment:
web_editor is auto_install, so it should be installed, I don't know if this is a problem for `web_responsive`

Reply:
Well, changing dependencies is not the ideal indeed...